packages/devtools/test/docker/__snapshots__/compose.test.ts.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`docker/compose serializeDockerComposeSpec should create a valid compose spec if called with basic services 1`] = `
+"version: '3.9'
+services:
+  redis:
+    image: redis:latest
+  codebase:
+    build: .
+    command: pnpm build
+    volumes:
+      - ./packages:/app/packages
+  postgres:
+    image: postgres
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+volumes:
+  pgdata: null
+  somevolume:
+    name: somevolumename
+"
+`;
+
+exports[`docker/compose serializeDockerComposeSpec should create a valid compose spec if called with no services 1`] = `
```

Comment:
What does the 1 stand for?

Reply:
It's a `jest` convention for identifying snapshots, purely internal to `jest`